docs/folder-structure.md

```diff
@@ -0,0 +1,96 @@
+# F36 Folder Structure
+
+To make it easy to maintain our files and folders we ask every contributor to follow this folder structure.
+Every package has its own folder, except for components which should be kept in the folder `/components` (every component must have its own folder).
+
+## Table of contents
+
+- [Components](#components)
+  - [Packages with more than one component](#packages-with-more-than-one-component)
+  - [Packages with utility functions](#packages-with-utility-functions)
+  - [Why do we use MDX?](#why-do-we-use-mdx)
+  - [Why is every `.mdx` file named "README"](#why-is-every-mdx-file-named-readme)
+
+## Components
+
+The basic folder structure of a component looks like this:
```

Comment:
maybe also point out that the `yarn generate component` would already create the folder with the files in the correct place?

Reply:
good point!